pom.xml

```diff
@@ -2,37 +2,249 @@
+      <id>mmartinadanx</id>
+      <name>Miguel Ángel Martín</name>
+      <organization>Ixxus</organization>
+      <organizationUrl>http://www.ixxus.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>mteodori</id>
+      <name>Marcello Teodori</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>nazarethjim</id>
+      <name>Nazareth Jimenez Vela</name>
+      <organization>Ixxus</organization>
+      <organizationUrl>http://www.ixxus.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>pmartinezga</id>
+      <name>Pablo Martinez Garcia</name>
+      <organization>Ixxus</organization>
+      <organizationUrl>http://www.ixxus.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>popovicsandras</id>
+      <name>András Popovics</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>robfrank</id>
+      <name>Roberto Franchini</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>zoltanpalfi</id>
+      <name>Zoltan Palfi</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>CTI777</id>
+      <name>Illia Goncharov</name>
+      <organization>IntroPro Ventures</organization>
+      <organizationUrl>http://introproventures.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>EquinoxBlack</id>
+      <name>Rodion Savchuk</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+    <developer>
+      <id>VitoAlbano</id>
+      <name>Vito Albano</name>
+      <organization>Alfresco</organization>
+      <organizationUrl>http://alfresco.com</organizationUrl>
+    </developer>
+  </developers>
+  <scm>
+    <url>http://github.com/Activiti/${project.artifactId}</url>
+    <connection>scm:git:git://github.com/Activiti/${project.artifactId}.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/Activiti/${project.artifactId}.git</developerConnection>
+  </scm>
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/Activiti/Activiti/issues</url>
+  </issueManagement>
+  <ciManagement>
+    <system>Travis</system>
+    <url>https://travis-ci.org/Activiti/${project.artifactId}</url>
+  </ciManagement>
   <properties>
-    <project.scm.repository>activiti-cloud-build</project.scm.repository>
-    <!-- versions -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <java.version>8</java.version>
+    <maven.compiler.source>${java.version}</maven.compiler.source>
```

Comment:
1.8 is expected here otherwise other plugins like checkstyle do not work, this should be amended like here:
https://github.com/Activiti/Activiti/blob/develop/pom.xml#L187-L191

Reply:
Fixed